Upstream: kind of, at least for ExtUtils-MakeMaker, don't know when it'll
          get pulled in by perl (currently at 7.24, fix is in 7.25_02)
Source: https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/commit/58fa199a752913dfb0455ac1397e897afcc1f9b4
Reason: Required for perl-Prevent-directory-chmod-race-attack.patch

From 9fd2fef597981144cfe03a76e55806888198b7fc Mon Sep 17 00:00:00 2001
From: James E Keenan <jkeenan@cpan.org>
Date: Mon, 8 May 2017 20:44:42 -0400
Subject: [PATCH] Correct the order of tests of chmod().

Per code review by haarg, the order of tests was wrong in the first place.
Hence, correctly re-ordering them is a better repair than changing one test's
description.

For: https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/pull/294
---
 cpan/ExtUtils-MakeMaker/t/eu_command.t | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/cpan/ExtUtils-MakeMaker/t/eu_command.t b/cpan/ExtUtils-MakeMaker/t/eu_command.t
index 269aa5c9..32a6f59d 100644
--- a/cpan/ExtUtils-MakeMaker/t/eu_command.t
+++ b/cpan/ExtUtils-MakeMaker/t/eu_command.t
@@ -151,20 +151,21 @@ BEGIN {
         is( ((stat('testdir'))[2] & 07777) & 0700,
             0100, 'change a dir to execute-only' );
 
-        # change a dir to read-only
-        @ARGV = ( '0400', 'testdir' );
+        # change a dir to write-only
+        @ARGV = ( '0200', 'testdir' );
         ExtUtils::Command::chmod();
 
         is( ((stat('testdir'))[2] & 07777) & 0700,
-            0400, 'change a dir to read-only' );
+            0200, 'change a dir to write-only' );
 
-        # change a dir to write-only
-        @ARGV = ( '0200', 'testdir' );
+        # change a dir to read-only
+        @ARGV = ( '0400', 'testdir' );
         ExtUtils::Command::chmod();
 
         is( ((stat('testdir'))[2] & 07777) & 0700,
-            0200, 'change a dir to write-only' );
+            0400, 'change a dir to read-only' );
 
+        # remove the dir we've been playing with
         @ARGV = ('testdir');
         rm_rf;
         ok( ! -e 'testdir', 'rm_rf can delete a read-only dir' );
